dts/arm/st/g4/stm32g4.dtsi

```diff
@@ -176,6 +176,13 @@
 			label = "LPUART_1";
```

Comment:
Move this commit before board as you configure it in board commit

Reply:
Could you clarify when you mean by this comment?

Reply:
Board configuration will use code provided in drivers commits.
So board commit should be the last one in this series.

Reply:
It is the last commit on my branch

Reply:
Ok, it doesn't appear in that order, this is a github problem then.